src/test/java/hudson/plugins/jira/JiraSiteSecurity1029Test.java

```diff
@@ -26,11 +30,6 @@
 import jenkins.model.Jenkins;
 import jenkins.security.ApiTokenProperty;
 import net.sf.json.JSONObject;
-import org.eclipse.jetty.ee9.servlet.DefaultServlet;
-import org.eclipse.jetty.ee9.servlet.ServletContextHandler;
-import org.eclipse.jetty.ee9.servlet.ServletHolder;
```

Comment:
No more references to a specific EE version, making this test robust against future EE upgrades (including EE 10).